doc/backup_recovery.rst

```diff
@@ -0,0 +1,130 @@
+.. _backup_recovery:
+
+Recovery from backups
+=====================
+
+In case of emergency ... break glass.
```

Comment:
Can we have a rough estimate "what is going to happen now" here?

Reply:
Updated